packages/build/src/log/messages/config.js

```diff
@@ -57,9 +57,10 @@ const INTERNAL_FLAGS = [
   'timeline',
   'explicitSecretKeys',
   'edgeFunctionsBootstrapURL',
+  'eventHandlers',
 ]
 const HIDDEN_FLAGS = [...SECURE_FLAGS, ...TEST_FLAGS, ...INTERNAL_FLAGS]
-const HIDDEN_DEBUG_FLAGS = [...SECURE_FLAGS, ...TEST_FLAGS]
+const HIDDEN_DEBUG_FLAGS = [...SECURE_FLAGS, ...TEST_FLAGS, 'eventHandlers']
```

Comment:
Any reason to hide this from the debug view? I think it could be useful when troubleshooting.

Reply:
So this was due to the functions not being able to be parsed by the YAML method underneath. Didn't dig too deep, but it was something in the visualisation it wasn't happy with it being a function. 

Good shout on it being useful, may show the entries + description here